code/obj/machinery/computer/syndicate/podbay.dm

```diff
@@ -0,0 +1,124 @@
+// Riot computer but reflavored for the Cairngorm
+
+/obj/machinery/computer/battlecruiser_podbay
+	name = "podbay authorization"
+	icon_state = "drawbr"
+	density = FALSE
+	glow_in_dark_screen = TRUE
+	/// How many authentications are needed to release the shielding on the podbay. Set the first time someone clicks this.
+	var/auth_need = null
+	var/list/authorized
+	desc = "Controls access to the podbay. Use this when your team is ready to go."
+
+	light_r = 1
+	light_g = 0.3
+	light_b = 0.3
+
+	/// Whether or not the podbay on the Cairngorm is authorized.
+	var/authed = FALSE
+	/// How long until we take matters into our own hands.
+	var/auth_delay = 10 MINUTES
+	var/area/battlecruiser_area
+
+	New()
+		..()
+		START_TRACKING
+
+	initialize()
+		battlecruiser_area = get_area_by_type(/area/syndicate_station/battlecruiser)
+		SPAWN(auth_delay)
+			authorize() // If they haven't done it before auth_delay, do it for em
+		..()
+
+	disposing()
+		src.authorized = null
+		STOP_TRACKING
+		..()
+
+	proc/authorize()
+		if(src.authed)
+			return
+
+		logTheThing(LOG_STATION, usr, "authorized Cairngorm podbay access")
+		authed = 1
+		src.ClearSpecificOverlays("screen_image")
+		src.icon_state = "drawbr-alert"
+		src.UpdateIcon()
+
+		for (var/obj/forcefield/battlecruiser/S in battlecruiser_area)
+			qdel(S)
+			LAGCHECK(LAG_REALTIME)
+
+		SPAWN(0.5 SECONDS)
+			// WAKE UP!!!!!
+			var/operative_datums = get_all_antagonists(ROLE_NUKEOP) + get_all_antagonists(ROLE_NUKEOP_COMMANDER)
+			var/operative_mobs = list()
+			for (var/datum/antagonist/operative in operative_datums)
```

Comment:
```suggestion
			for (var/datum/antagonist/operative as anything in (get_all_antagonists(ROLE_NUKEOP) + get_all_antagonists(ROLE_NUKEOP_COMMANDER)))
```
`var/operative_datums` is only used once and could be replaced by the above.

Reply:
Hmm ok. I'll add a comment touching on what it does then, just for clarity.